client/src/models/index.js

```diff
@@ -0,0 +1,12 @@
+var express = require('express');
+var path = require('path');
+var router = new express.Router();
+
+
```

Comment:
Are these blank lines here for something?

Reply:
they were already there I just moved the file

Reply:
Fair enough, would you mind deleting them?

Reply:
ok no problem

Reply:
this is the same file Fiona picked up, it's deleted now